EN/system/str_jp/stringfile_battlevoice.csv

```diff
@@ -1,36 +1,36 @@
-battlevoice_chr000_100,I hope this proves at least something of a fight.,
+battlevoice_chr000_100,I hope this proves at least somewhat of a fight.,
 battlevoice_chr000_101,I don't wanna look at you.,
 battlevoice_chr000_102,"Sure, I'll take you on.",
 battlevoice_chr000_103,Hm... This could be fun.,
-battlevoice_chr000_104,Are you joking? Come back in a hundred years.,
+battlevoice_chr000_104,"You're joking, right? Try again in a hundred years.",
 battlevoice_chr000_105,I'll have you take responsibility for killing me.,
 battlevoice_chr000_106,Okay! I'll take you on for a bit!,
 battlevoice_chr000_107,No way! I'd never lose my temper like that!,
-battlevoice_chr000_108,I don't have time to dally around.,
-battlevoice_chr000_109,I gotta look out for myself? No problem!,
+battlevoice_chr000_108,I don't have time to dilly-dally.,
+battlevoice_chr000_109,I gotta look out for myself? No problemo!,
 battlevoice_chr000_650,"You've got guts, I'll give you that.",
 battlevoice_chr000_651,"Alright, how about we play around for real?",
-battlevoice_chr000_652,Maybe I should be a little serious?,
+battlevoice_chr000_652,Maybe I get a little serious?,
 battlevoice_chr000_653,You're gonna fight me with that equipment? Looking down on me?,
 battlevoice_chr000_630,Really think this is even gonna be a fight?,
 battlevoice_chr000_631,"Yet again, your body lacks class.",
 battlevoice_chr000_632,You've become too ingrained into this land.,
 battlevoice_chr000_110,"Alright, it's on!",
 battlevoice_chr000_111,Don't say I didn't warn you!,
-battlevoice_chr000_112,Seems I won't have to hold back...,
-battlevoice_chr000_120,Let's keep going!,
+battlevoice_chr000_112,Looks like I won't have to hold back.,
+battlevoice_chr000_120,Let's keep at it!,
 battlevoice_chr000_121,You want to try that again?,
 battlevoice_chr000_122,I expected better.,
 battlevoice_chr000_637,I expected better.,
 battlevoice_chr000_123,Still wanna go?,
 battlevoice_chr000_663,Still wanna go?,
-battlevoice_chr000_124,"Yeah, that was fun, Shiki!",
-battlevoice_chr000_125,Get up. I've not had enough yet.,
+battlevoice_chr000_124,"Yup, that was a lotta fun, Shiki!",
+battlevoice_chr000_125,Get up. I haven't had enough yet.,
 battlevoice_chr000_620,Oh? Are you feeling bad?,
 battlevoice_chr000_660,Get up already. I'm sick of waiting.,
 battlevoice_chr000_661,"Just a little more, I think.",
-battlevoice_chr000_662,"Too slow. You did gain weight, didn't you?",
-battlevoice_chr000_635,Playing dead? It suits you.,
+battlevoice_chr000_662,"Too slow. You DID gain weight after all!",
+battlevoice_chr000_635,Playing dead? Suits you to a T.,

```

Comment:
Arcueid uses her 'serious' voice against Roa, so maybe "Playing dead? Fitting." or something is more appropriate.